docs/fundamentals/executor/hub/use-hub-executor.md

```diff
@@ -153,3 +153,11 @@ jina hub pull jinahub://<UUID>[:<SECRET>][/<TAG>]
 
 
 The source code of the Executor will be stored at `~/.jina/hub-packages`.
+
+### list the local location of executors
```

Comment:
For headings, ensure:

- Top level headings (`# foo`) - [Title Case](https://titlecaseconverter.com/#TitleCapitalizationRules)
- Lower level headings (`## bar`, `### baz`) - [Sentence case](https://titlecaseconverter.com/#WhatIsSentenceCase)

Links above go to a tool that will auto-format your text accordingly.

We use AP style

Reply:
Thanks for the suggestion, I will fix it.